templates/environment/partials/custom-resources.yml

```diff
@@ -1,14 +1,15 @@
 EnableLongARNFormatAction:
+  Metadata:
+    'aws:copilot:description': 'Custom resource enabling long ARN formats for the authenticated AWS principal'
```

Comment:
Maybe we can remove the "Custom resource" prefix and write it like this to keep it concise:
```
Request and validate an ACM certificate for your domain
```